modules/admin_manual/pages/configuration/general_topics/search.adoc

```diff
@@ -67,32 +67,103 @@ For the URL, both `HTTP` and `HTTPS` incuding the address and port can be used.
 
 === Search External Storage
 
 Define if external storage should be included in ES indexing by setting the checkmark accordingly with btn:[Scan external Storages]. Setting this checkmark not only enables search in external storages, but also search in federated shares. Note that this setting requires to rebuild the index.
 
+=== Connector Setup
+
+There are 2 different indexes. The `Legacy` is the old / current one while the `RelevanceV2` is the new one. The difference between the two is how index data is stored because both provide different capabilities and index in different ways. This results in different search capabilities. The Legacy doesn't differ from the current search while the RelevanceV2 has new capabilities.
+
+To populate the new connector with enhanced capabilities, an occ command needs to be run for each user, creating index data accordingly. See the occ command section xref:configuration/server/occ_command.adoc#fill-a-secondary-index[Fill a Secondary Index] for more details.
+
+image::apps/search_elastic/connector_setup.png[Connector Setup,width=600]
+
+==== Migrating to the RelevanceV2 Connector:
+
+* If you haven't indexed anything yet, you are encouraged to setup the connectors you want to use as part of the app configuration. The recommended one is `RelevanceV2` for `write` and `search`.
+
+* If you have indexed data already, use the following steps to migrate to the new index scheme:
+
+. Assuming you have the `Legacy` connector setup for `write` and `search`.
+
+. Add the `RelevanceV2` connector to the list of `write` connectors. The list should have both `Legacy` and `RelevanceV2`.
+
+. Run the `occ search:index:fillSecondary RelevanceV2 <user>` xref:configuration/server/occ_command.adoc#fill-a-secondary-index[command]. The command needs to be run for all  users, or at least the ones using the search app. Note that this step is expected to take a lot of time.
+
+. Once indexed data has been migrated for all users, you can switch the search connector to use the new `RelevanceV2` search capabilities.
+
+. After checking everything is good, you can remove the old `Legacy` connector from the list of write connectors.
+
+. Finally you can completely remove the old index from Elasticsearch.
+
+With step 2, you will be writing into both indexes at the same time. This is expected to be slower. Note that step 2 just takes care of new files. Files indexed previously won't be present in the new index. This is why step 3 is there.
+
+Step 4 is important and you should stop at that point for a while. If something goes wrong, you can still revert things, in particular, you can switch back to the `Legacy` connector. In this case, use the occ command referenced above with the `Legacy` index.
+
+From step 5 the actions are irreversible. If you want to go back, you'll have to start a new migration.
```

Comment:
Shouldn't these explanations be mentioned right with the steps not after the users may have performed them already?